lib/matplotlib/text.py

```diff
@@ -200,7 +200,9 @@ def contains(self, mouseevent):
         In the case of text, a hit is true anywhere in the
         axis-aligned bounding-box containing the text.
 
-        Returns True or False.
+        Returns
+        -------
+        bool : bool
```

Comment:
with returns, you don't necessarily have to use this syntax. You could also write:

```
Returns
-------
bool
```